.github/instructions/agent-commit-instructions.md

```diff
@@ -0,0 +1,69 @@
+---
+exclude-agent: "code-review"
+---
+
+# Commit Instructions for GitHub Copilot Agent
+
+This file provides instructions for the GitHub Copilot coding agent when making commits to this repository.
+
+## Commit Message Format
+
+All commits **must** follow the Conventional Commits standard:
+
+```
+<type>(<scope>): <short description>
+```
+
+### Required Components
+
+- **type**: The kind of change (required)
+- **scope**: Single-word identifier for the affected subpackage/domain (optional but recommended)
+- **description**: Brief summary in lowercase, no ending punctuation (required)
+
+## Commit Types
+
+- **feat**: New feature
+- **fix**: Bug fix
+- **docs**: Documentation changes
+- **refactor**: Code changes that neither fix bugs nor add features
+- **test**: Adding or updating tests
+- **chore**: Maintenance tasks
+- **style**: Code style/formatting changes
+- **perf**: Performance improvements
+- **build**: Build system or dependency changes
+- **ci**: CI configuration changes
+- **revert**: Revert previous commit
+
+## Scope Guidelines
+
+Use a **single word** identifying the primary affected subpackage or domain:
+
+Common scopes: `api`, `apiserver`, `cli`, `storage`, `model`, `controller`, `agent`, `database`, `cmd`, `core`, `cloud`
+
+Omit scope only if no clear primary domain applies.
+
+## Examples
+
+```
+feat(api): add user authentication endpoint
+```
+
+```
+fix(storage): resolve race condition in volume attachment
+```
+
+```
+docs: update contributing guidelines
```

Comment:
This is an example of a bad commit message as it is too broad.

I'd prefer we give examples that clarify the change with more detail.

```suggestion
docs: add CLA requirements to contributing guidelines
```